### PR TITLE
close re-render/etc issues if nothing is needed

### LINKED ISSUE #226: Close re-render issue on no change
If requesting a re-rendering in an issue is a no change (e.g. https://github.com/conda-forge/dask-ndmeasure-feedstock/issues/7 ), the bot should feel free to close the issue (as it is resolved).

### PR DESCRIPTION
**Pull request by dougalsutherland**
Fixes #226.

### CODE REVIEW

**Comment by jakirkham** [comment]
LGTM. Thanks @dougalsutherland.

Not sure what happened to CI. So went ahead and restarted it.

**Comment by dougalsutherland** [comment]
I just checked and tests pass locally, but the Travis build blew up again. Will look into it a little later... :confused:

**Comment by dougalsutherland** [comment]
Example failure: https://github.com/conda-forge/conda-forge-webservices/pull/54#issuecomment-412106931

It's a `conda-smithy` issue caused by a new `ruaml.yaml` version; started happening to me locally after updating 
```
    ruamel.yaml: 0.15.50-py36_0         conda-forge --> 0.15.52-py36_0         conda-forge
    ruamel_yaml: 0.15.46-py36h470a237_0 conda-forge --> 0.15.52-py36h470a237_0 conda-forge
```
(Not sure which of the two was actually in use before, because of that gross situation.)

Will fix on `conda-smithy`.

File: conda-requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 conda-smithy>=3
+ruamel.yaml==0.15.51  # until https://github.com/conda-forge/conda-smithy/pull/871
```
Comment (isuruf) [inline conda-requirements.txt]: @dougalsutherland, does `0.15.51` work? We may have to pin it back to `0.15.50`

**Pull request merged**